src/templates/Plain.js

```diff
@@ -52,7 +52,6 @@ export default function Plain({ data }) {
                     <MDXRenderer images={images}>{body}</MDXRenderer>
                 </MDXProvider>
             </article>
-            <Section />
```

Comment:
@smallbrownbike Git says you added this line, as far as I can tell it doesn't do anything? Seems to just quietly add some extra margin

Reply:
No reason for that to be there. It may have been added by accident!